src/processor/operator/ddl/create_rel_table.cpp

```diff
@@ -6,7 +6,7 @@ namespace processor {
 string CreateRelTable::execute() {
     auto newRelTableID = catalog->addRelTableSchema(
         tableName, relMultiplicity, propertyNameDataTypes, srcDstTableIDs);
-    relsStatistics->addTableStatistic(catalog->getWriteVersion()->getRelTableSchema(newRelTableID));
+    relsStatistics->addTableStatistic(catalog->getWriteVersion()->getTableSchema(newRelTableID));
```

Comment:
One thing worth noticing here is you don't need to cast when adding relStatistics but do need cast for nodeStatistics. This might be desired behaviour but worth double checking

Reply:
To be safe, revert to `getReltableSchema()`